src/components/modules/tools.ts

```diff
@@ -221,7 +221,30 @@ export default class Tools extends Module {
     /**
      * Assign internal tools
      */
-    this.config.tools = _.deepMerge({}, this.internalTools, this.config.tools);
+    this.config.tools = {
+      ...this.internalTools,
+      ...this.config.tools,
+      bold: {
```

Comment:
I don't think it is good idea to hardcode internal tools here. It would better to store their names in one place